e2e/cypress/integration/integrations/incoming_webhook/description_length_check_spec.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @integrations
+
+let testTeam;
+
+describe('Integrations', () => {
+    const maxDescription = '12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890';
+
+    before(() => {
+        // # Login as test user and visit the newly created test channel
+        cy.apiInitSetup().then(({team}) => {
+            testTeam = team;
+
+            // # Visit the Incoming Webhooks add page
+            cy.visit(`/${team.name}/integrations/incoming_webhooks/add`);
+        });
+    });
+
+    it('MM-T636 Description field length check', () => {
+        // * Check incoming description field only accepts 500 characters
+        cy.get('#description').clear().type(`${maxDescription}123`).should('have.value', maxDescription);
```

Comment:
Please use suggested constant above to not repeat the same hard-coded value.
```suggestion
        cy.get('#description').clear().type(overMaxDescription).should('have.value', maxDescription);
```